pilz_robot_programming/examples/demo_brake_test_program.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
```

Comment:
catkin_lint reports:
```
pilz_robot_programming: warning: file 'examples/demo_brake_test_program.py' is executable but not installed
```
=> Add `demo_brake_test_program.py` to `CMakeLists.txt` [here](https://github.com/PilzDE/pilz_industrial_motion/blob/4e311ce223ce6c4a1bb3c3e2bb2299970ecfcf67/pilz_robot_programming/CMakeLists.txt#L14).

Reply:
Done